pkgs/development/python-modules/playwright/default.nix

```diff
@@ -46,10 +48,13 @@ buildPythonPackage rec {
     git commit -m "workaround setuptools-scm"
 
     substituteInPlace setup.py \
-      --replace "greenlet==2.0.1" "greenlet>=2.0.1" \
-      --replace "pyee==8.1.0" "pyee>=8.1.0" \
-      --replace "setuptools-scm==7.0.5" "setuptools-scm>=7.0.5" \
-      --replace "wheel==0.38.1" "wheel>=0.37.1"
+      --replace "setuptools-scm==8.0.4" "setuptools-scm" \
+      --replace "wheel==0.41.2" "wheel"
+
+    substituteInPlace pyproject.toml \
+      --replace 'requires = ["setuptools==68.2.2", "setuptools-scm==8.0.4", "wheel==0.41.2", "auditwheel==5.4.0"]' \
+                'requires = ["setuptools", "setuptools-scm", "wheel", "auditwheel"]' \
+      --replace 'version_file = "playwright/_repo_version.py"' ""
```

Comment:
I see now, `setuptools_scm` 7.x doesn't support `version_file`. I wrote a cursed overlay that updates everything and it works for me now:
```nix
(final: prev: {
  python3 = prev.python3.override {
    packageOverrides = finalPkgs: prevPkgs: {
      playwright = (prevPkgs.playwright.override (prevArgs: {
        setuptools-scm = prevArgs.setuptools-scm.overridePythonAttrs (prevAttrs: {
          version = "8.0.4";
          src = prevPkgs.fetchPypi {
            pname = "setuptools-scm";
            version = "8.0.4";
            hash = "sha256-tfQ/9oAGaVlRk/0JiRVk7p0dfcsZbKtLJQbVOi4clcc=";
          };
        });
      })).overridePythonAttrs (prevAttrs: {
        postPatch = builtins.replaceStrings [ ''--replace 'version_file = "playwright/_repo_version.py"' ""'' ] [ "" ] prevAttrs.postPatch;
      });
    };
  };
})
```
It _seems_ like setuptools 8.x is fairly compatible with 7.x, so it might be possible to update it everywhere.